src/lib.rs

```diff
@@ -2,6 +2,7 @@
 #![allow(clippy::missing_safety_doc)]
 #![allow(clippy::module_inception)]
 #![allow(clippy::too_many_arguments)]
+#![feature(vec_into_raw_parts)]
```

Comment:
Would it be possible to find a workaround to avoid introducing this unstable feature ?

Reply:
Yep, less pretty but works :)